content/katas/_index.md

```diff
@@ -98,10 +98,148 @@ rotate("hello",-2) => lohel
 rotate("hello",-3) => llohe
 ```
 
+## Print (triangles & squares) using loops
+
+**1.**
+
+Write a function that takes in number as argument and prints a square
+```
+// JavaScript
+
+let square(n) => 
+expected output: remember to use "*" 
+
+#####
+#####
+#####
+#####
+#####
+```
+**2.**
+ Write a function that takes in number as argument and draws a right angled triangle  using the " * " characters. If the input is 4, then the output is a traingle with height 4
+ 
+```
+// JavaScript
+let triangle(4) => 
+expected output: RIGHT ANGLE TRIANGLE
+ 
+ *
+ **         
+ ***              
+ ****
+```
+
+**3.**
+Write a function that takes in number as argument and draws a reflection/mirror of right angled triangle above using the " * " characters . If the input is 4, then the output is a traingle with height 4
+```
+// JavaScript
+let triangle(5) => 
+expected output: MIRROR OF RIGHT ANGLE TRIANGLE
+
+   *
+  **              
+ ***
+****
+```
+**4.**
+Write a function that takes in number as argument and draws a Hollow right angled triangle above using the " * " characters . If the input is 4, then the output is a traingle with height 4
+
+```
+// JavaScript
+let triangle(5) => 
+expected output: HOLLOW RIGHT ANGLE TRIANGLE AND FOR ITS MIRROR.      
+
+*
+* *
+*  *
+*   *
+******
+```
+**5.**
+Write a function that takes in number as argument and prints a hollow square in the center
+```
+// JavaScript
+let square(n) => 
+expected output: remember to use "*" 
+#####
+#   #
+#   #
+#   #
+#####
+```
+**6.** 
+Write a function that takes in number as argument and draws a pyramid triangle above using the " * " characters . If the input is 4, then the output is a traingle shaped as pyramid with height 4
+```
+// JavaScript
+let triangle(5) => 
+expected output: PYRAMID TRIANGLE.      
+    
+    *
+   ***
+  *****
+ *******  
+*********
+```
+
+**7.**
+Write a function that takes in number as argument and draws a Hollow pyramid triangle above using the " * " characters . If the input is 4, then the output is a traingle shaped as pyramid with height 4
+```
+// JavaScript
+let triangle(5) => 
+expected output: HOLLOW RIGHT ANGLE TRIANGLE AND FOR ITS MIRROR.      
+    
+    *
+   * *
```

Comment:
Ok, we need to do another git feature branching workshop. You are mixing your stuff up.

I need you to know how to use git like a professional before I can consider you competent. I need you to demonstrate your competence